lib/utils.pm

```diff
@@ -1052,7 +1052,10 @@ some basic logging information to the serial output.
 sub handle_emergency {
     if (match_has_tag('emergency-shell')) {
         # get emergency shell logs for bug, scp doesn't work
+        type_password;
+        send_key 'ret';
         script_run "cat /run/initramfs/rdsosreport.txt > /dev/$serialdev";
+        script_run "journalctl --no-pager -o short-precise > /dev/$serialdev";
```

Comment:
Do we also want to add dmesg? Or do we rely on serial output here?
Also, I believe that some sort of delimiter between those calls would be helpful, as output might be considerably long.

Reply:
I think it can be useful. I could modify it to save logs from last 15 minutes or so. WDYT? 
I added a delimeter, even though the journalctl logs start like:
-- Logs begin at Fri 2020-11-13 10:58:20 EST, end at Fri 2020-11-13 10:59:52 EST. --

Reply:
I believe that it's better not to limit it to last 15 minutes, unless we find any issue with that.